lib/screens/forgot_password/cubit/forgot_password_state.dart

```diff
@@ -12,20 +12,22 @@ class ForgotLoading extends ForgotPasswordState {}
 
 class ForgotEmailInitial extends ForgotPasswordState {}
 
-class ForgotEmailFailure extends ForgotPasswordState {}
-
 class ForgotOTPInitial extends ForgotPasswordState {}
 
-class ForgotOTPFailure extends ForgotPasswordState {}
+class ForgotOTPFailure extends ForgotPasswordState {
```

Comment:
Since Error is used everywhere as a convention now.
```suggestion
class ForgotPasswordError extends ForgotPasswordState {
```